nvflare/fuel/f3/drivers/prefix.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import struct
+from dataclasses import dataclass
+
+from nvflare.fuel.f3.comm_error import CommError
+from nvflare.fuel.f3.connection import BytesAlike
+
+PREFIX_STRUCT = struct.Struct(">IHBBHHHH")
+PREFIX_LEN = PREFIX_STRUCT.size
+
+
+@dataclass
+class Prefix:
+    """Prefix is the 16-byte fixed header for the SFM frame, every frame must have this prefix.
+    Beside all the other attributes, it provides framing for the message. Framing is needed if the
+    frame is sent over byte streams like TCP or sockets.
+
+    The 8 fields in the prefix are all integers encoded in big-endian,
```

Comment:
if this class is for SFM, should this be put inside the SFM folder?

Reply:
Good idea. Moved.